newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/index.js

```diff
@@ -25,6 +27,7 @@ import { type MessageDescriptor } from '../../Utils/i18n/MessageDescriptor.flow'
 export default ({
   internalName: 'LocalFile',
   name: t`Local file system`,
```

Comment:
We could even rename this to "Your computer". Simple, understandable, obvious.

Reply:
Yep, that's a good idea